### PR TITLE
feat(cli): New Account Command

### LINKED ISSUE #164: CLI Quota Command
**Describe the solution you'd like**
I would like there to be a command like `$ censys quota` that returns your current quota printed nicely.

**Describe alternatives you've considered**
```sh
$ censys quota
Email: test@censys.io
Used: 2
Allowance: 250
Resets At: 12:00 am on Nov 1st, 2021

$ censys quota --json
{"used": 2, "allowance": 250, "resets_at": "2021-10-01 12:00:00"}
```

### PR DESCRIPTION
**Pull request by thehappydinoa**
- Added `censys account` command

Closes #164

### CODE REVIEW

File: censys/cli/commands/account.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        if args.json:
+            print_json(data=account)
+        else:
+            table = Table("Key", "Value", show_header=False, box=box.SQUARE)
+            table.add_row("Email", account["email"])
+            table.add_row("Login", account["login"])
+            table.add_row("First Login", account["first_login"])
+            table.add_row("Last Login", account["last_login"][:-7])
+            quota = account["quota"]
+            table.add_row("Quota", f"{quota['used']} / {quota['allowance']}")
```
Comment (mbremerkamp) [inline censys/cli/commands/account.py:30]: Would it be slightly more descriptive to use `Query Quota`?

File: censys/cli/commands/account.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    """
+    try:
+        client = CensysSearchAPIv2(args.api_id, args.api_secret)
+        account = client.account()
+        if args.json:
+            print_json(data=account)
+        else:
+            table = Table("Key", "Value", show_header=False, box=box.SQUARE)
+            table.add_row("Email", account["email"])
+            table.add_row("Login", account["login"])
```
Comment (mbremerkamp) [inline censys/cli/commands/account.py:26]: Thoughts on using `Login ID`?

**Review by mbremerkamp — APPROVED** [review]
The table looks great.

File: censys/cli/commands/account.py (hunk 2 above)
Comment (thehappydinoa) [reply censys/cli/commands/account.py]: Big fan, just updated

**Pull request merged**